File: 153. Find Minimum in Rotated Sorted Array.py
class Solution(object):
    def findMin(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        minn = nums[0]
        for i in xrange(1, len(nums)):
            if nums[i] < minn:
                return nums[i]
        return minn

File: 153. Find Minimum in Rotated Sorted Array.cpp
class Solution {
public:
    int findMin(vector<int>& nums) {
        int min = nums[0];
        for (int i = 1; i < nums.size(); i++)
        {
            if (nums[i] < min) return nums[i];
        }
        return min;
    }
};
